### DISCUSSION TITLE
Feature request: Tom explorer favourites / pinned items

### DISCUSSION THREAD

**Discussion by samaguire** [TabularEditor/TabularEditor3#1101] [Ideas]
I thought it might be handy to have the functionality to favourite / pin objects in the TOM explorer for quick navigation, i.e. jumping the active selection to the selected favourited / pinned object in the TOM explorer window.

Use case would be when working with a group of tables from within a large model. This would help with avoid scrolling through the object list when jumping between table columns; for example, when checking column properties.

I though the favourited / pinned items could list in a new panel/section just above the TOM explorer.

**Comment by otykier** [comment]
That’s an excellent suggestion. There’s some design work involved they we’ll have to figure out, but I really like it.

Until we have this feature available, a possible (albeit cumbersome) alternative is to create a diagram, add your favorite tables and then arrange them in a compact way. When you click on a table in the diagram view, the TOM Explorer will also navigate to that table.

**Comment by mlonsk** [comment]
Table Groups in Tabular Editor 3 partially solves this feature request, and I will close it. Apologies for not updating here sooner.
